src/dll/Hooks/ExecuteProcess.cpp

```diff
@@ -0,0 +1,100 @@
+#include "ExecuteProcess.hpp"
+#include "Addresses.hpp"
+#include "App.hpp"
+#include "Hook.hpp"
+#include "Systems/ScriptSystem.hpp"
+#include "stdafx.hpp"
+
+namespace
+{
+bool isAttached = false;
+
+struct FixedWString
```

Comment:
This this struct used somewhere else too? I would prefer to have it in the SDK.

Reply:
It is, but seems to be limited to file/process operations - it looked similar to a couple structs (maybe even something in `std`?), but things didn't quite match up. It could be a template of some sort. There are a couple of functions that seem to deal the type specifically.

`pattern 48 89 5C 24 08 48 89 74 24 10 57 48 83 EC 20 33 F6 48 8B FA 48 89 31 48 8B D9 48 89 71 08 48 85` is one of them.

If you have better names, I'm all ears :)

Reply:
I am also not sure how to name it, seems like a string with a max capacity 🤔 But if needed we can move it to the SDK later.

Reply:
I'll move it to a header for now, and when I have a chance to reverse it more with its methods, we can add it to the SDK :)